compiler/p/codegen/TreeEvaluatorVMX.cpp

```diff
@@ -96,7 +96,7 @@ TR::Register *OMR::Power::TreeEvaluator::arraysetEvaluator(TR::Node *node, TR::C
             else
                {
                generateTrg1ImmInstruction(cg, TR::InstOpCode::lis, node, fillReg, (int16_t)halfword);
-               generateTrg1Src1ImmInstruction(cg, TR::InstOpCode::ori, node, fillReg, fillReg, halfword);
+               generateTrg1Src1ImmInstruction(cg, TR::InstOpCode::ori, node, fillReg, fillReg, (uint16_t)halfword);
```

Comment:
No need to truncate. This value cannot be out-of-range.

Reply:
Yes, safe but redundant.